dev-site-config/navigation.config.js

```diff
@@ -41,6 +41,13 @@ const navConfig = {
       capabilities: {
         devTools: true,
       },
+    }, {
+      path: '/secondary-scroll',
```

Comment:
Can we add these examples to the `/test` tab?

Reply:
I think it makes sense to add a new tab because we're testing the framework of dev-site not the contents of the page. It's annoying that it breaks all our other screenshots, but it's very hard to isolate. I also want to keep all the filler content out of the tests page. And I don't want to have to update alot of screenshots when adding new content tests to the test page

Reply:
If we wanted to add it to any existing tab, the secondary navigation tab would be appropriate.
https://engineering.cerner.com/terra-dev-site/secondary-nav-test/terra-dev-site/themed

Reply:
If we want to scope it by framework test concept, should these be included in the secondary-nav tab then? We can include it as a subnav like:
```
/tests
   themed-test
    secondary-nav-test
   /secondary-nav
   /secondary-scroll
```

Reply:
Alrighty, I will remove the secondary-scroll tab and move the tests to the secondary nav

Reply:
Done: https://github.com/cerner/terra-dev-site/pull/217/commits/d4338a8acdbe3a03e4ec82ad6de5e57d13ec1708